src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceRepository.kt

```diff
@@ -10,8 +10,11 @@ import org.springframework.stereotype.Repository
 import java.time.LocalDateTime
 
 @Repository
-interface LicenceRepository : JpaRepository<Licence, Long>, JpaSpecificationExecutor<Licence> {
+interface LicenceRepository :
+  JpaRepository<Licence, Long>,
+  JpaSpecificationExecutor<Licence> {
```

Comment:
is this used at all?

Reply:
Added about 7 months back in the initial creation of the API - doesn't look like it is used - removed.